dynamic_rest/serializers.py

```diff
@@ -147,6 +149,18 @@ def __new__(cls, *args, **kwargs):
         if not issubclass(list_serializer_class, DynamicListSerializer):
             list_serializer_class = DynamicListSerializer
         meta.list_serializer_class = list_serializer_class
+
+        # If a lookup field has been defined in the serializer that
+        # field becomes the ID field for the serializer
+        lookup_field = getattr(meta, 'lookup_field', None)
+        if lookup_field is not None:
+            model = getattr(cls.Meta, 'model')
+            info = model_meta.get_field_info(model)
+            # XXX: This might not be the way to correctly solve this
+            cls._declared_fields['id'] = \
```

Comment:
This definitely seems a bit hacky; what is the significance of the field called "id"? Where in DRF/DREST is that used?